lib/fontcustom/generator/font.rb

```diff
@@ -58,6 +59,17 @@ def set_glyph_info
           end
         end
 
+        files.each do |file|
```

Comment:
This second iterator is unnecessary, move this logic in to the iterator above and change the conditional logic to handle File.symlink? within the same loop